db/driver_mock_test.go

```diff
@@ -0,0 +1,56 @@
+// Code generated by MockGen. DO NOT EDIT.
+// Source: driver.go
+//
+// Generated by this command:
+//
+//	mockgen -source=driver.go -destination=driver_mock_test.go -package=db
```

Comment:
1. Any changes we can put mocks in a `/mocks` package? `db/mocks/driver_mock.go`
2. Please add a makefile target that regenerates all mocks

Reply:
I believe Arash has had some objections to a common mocks package in the past so perhaps best to discuss with it.

Reply:
@adshmh Do you mind chiming in on this topic?

While we don't have to resolve it here & now, I want to document the TODO (if it applies) w/ reasoning & rationale.